Api.Flickr.Tests/Api.Flickr.Tests.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
We discuss that will use 2.2.

Reply:
Rider does not support lower version selection :(